spec/01-unit/04-prefix_handler_spec.lua

```diff
@@ -759,6 +759,7 @@ describe("NGINX conf compiler", function()
           prefix = tmp_config.prefix,
           proxy_listen = "127.0.0.1:8000 ssl",
           admin_listen = "127.0.0.1:8001 ssl",
+          ssl_cipher_suite = "custom",
```

Comment:
Maybe we can add a test like  https://github.com/Kong/kong/pull/6470/files#diff-1ec0492b7c203c5f8e7924d1451c236131217a17006fd33beacbb680871fe803R772. By setting `proxy_listen  = "127.0.0.1:8000 ssl",` and `nginx_http_ssl_dhparam` to a predefined group, the file should be generated and the `ssl_dhparam` directive should points to the correct file.